### PR TITLE
WT-2371: parent split cannot access the page after page-index swap

### PR DESCRIPTION
**Pull request by keithbostic**
@michaelcahill, @agorrod, for your review/consideration.

### CODE REVIEW

File: src/btree/bt_split.c [hunk 1]
```diff
@@ -875,16 +875,24 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref, WT_REF **ref_new,
...
+		WT_ERR(__wt_verbose(session, WT_VERB_SPLIT,
+		    "%p: reverse split into parent %p, %" PRIu32 " -> %" PRIu32
+		    " (-%" PRIu32 ")",
+		    ref->page, parent, parent_entries, result_entries,
+		    parent_entries - result_entries));
+	else
+		WT_ERR(__wt_verbose(session, WT_VERB_SPLIT,
+		    "%p: split into parent %p, %" PRIu32 " -> %" PRIu32
+		    " (+%" PRIu32 ")",
+		    ref->page, parent, parent_entries, result_entries,
```
Comment (agorrod) [inline src/btree/bt_split.c]: Won't ``ref->page`` always be NULL?

**Comment by michaelcahill** [comment]
Thanks, @keithbostic, lgtm.  I'll merge.

File: src/btree/bt_split.c (hunk 1 above)
Comment (keithbostic) [inline src/btree/bt_split.c]: Yes, it will -- do you think it's worth fixing?

Comment (agorrod) [inline src/btree/bt_split.c]: Nope.

**Pull request merged**